teleporters/init.lua

```diff
@@ -8,6 +8,12 @@ teleporters = {}
 local PLAYER_COOLDOWN = 0.5
 local going_up_effect = true
 -- end config
+local efectsound=""
+if minetest.get_modpath("mcl_sounds") then
+efectsound=mcl_sounds.node_sound_stone_defaults()
+else
+efectsound=default.node_sound_stone_defaults()
+end
```

Comment:
Missing indentation and spaces, and `efectsound` does not need to be initialized 
```suggestion
local efectsound
if minetest.get_modpath("mcl_sounds") then
	efectsound = mcl_sounds.node_sound_stone_defaults()
else
	efectsound = default.node_sound_stone_defaults()
end
```